esy.json

```diff
@@ -3,12 +3,13 @@
   "version": "7.2.0",
   "dependencies": {
     "ocaml": "4.12.0+BS",
-    "@esy-ocaml/reason": "^3.7.0",
+    "@opam/reason": "^3.7.0",
     "@opam/dune": "*",
     "@opam/dune-action-plugin": "*",
     "@opam/ocamlfind": "1.8.1"
   },
   "devDependencies": {
+    "@opam/ocaml-lsp-server": "ocaml/ocaml-lsp:ocaml-lsp-server.opam",
```

Comment:
That's nice

Reply:
Oh, I accidentaly included that patch (oops). Anyway might be useful :)